### DISCUSSION TITLE
Example configuration using SASL_SSL and JAAS?

### DISCUSSION THREAD

**Discussion by erizzo** [provectus/kafka-ui#1103] [Q&A] [answered]
Is there any example of Kerberos authentication, using SASL_SSL as the protocol and JAAS (with a keytab file)? We have a Spring Cloud Stream app configured that way (set up by someone else) so I have all the necessary values; I just need to know how what config properties to use for kafka-ui to connect to the same cluster.

Thanks,
Eric

**Comment by Haarolean** [comment, marked as answer]
Hey, this might be what you're looking for.
https://github.com/provectus/kafka-ui/issues/858
